automation/jleechanorg_pr_automation/openai_automation/codex_github_mentions.py

```
@@ -124,15 +152,71 @@ async def connect_to_existing_browser(self) -> bool:
 
-            # Always create a new page to avoid browser UI elements (Omnibox, Extensions, etc.)
-            self.page = await self.context.new_page()
-            print("📄 Created new page for automation")
+            self.page = await self._select_existing_page()
+            if self.page:
+                print("📄 Reusing existing tab for automation")
+            else:
+                self.page = await self.context.new_page()
+                print("📄 Created new page for automation")
             return True
         except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)